### PR TITLE
Split military outposts and bunkers

### PR DESCRIPTION
**Pull request by krulunio**
#### Summary
None

#### Additional context
I plan to add those locations as possible base camp locations, having them under same OMTs makes my plan near impossible

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
If you change overmap ids you need to add migration code (e.g. #41910).

**Comment by krulunio** [comment]
Actually, I didn't change or remove old IDs, only added new ones.

**Comment by ZhilkinSerg** [comment]
> Actually, I didn't change or remove old IDs, only added new ones.

Right, it was just a warning for your awareness.

File: data/json/monstergroups/zanimal_upgrades.json [hunk 1]
```diff
@@ -21,6 +21,17 @@
...
+    "type": "monstergroup",
+    "name": "GROUP_ZOMBIE_PIG_UPGRADE",
+    "default": "mon_zombie_pig",
+    "//": "No bionics or fungal",
+    "monsters": [
+      { "monster": "mon_zombie_pig", "freq": 45, "cost_multiplier": 5 },
+      { "monster": "mon_zpig_brute", "freq": 45, "cost_multiplier": 2 },
+      { "monster": "mon_zombie_pig_gas", "freq": 45, "cost_multiplier": 2 }
+    ]
+  },
```
Comment (ZhilkinSerg) [inline data/json/monstergroups/zanimal_upgrades.json:34]:
```suggestion
```

File: src/worldfactory.cpp [hunk 2]
```diff
@@ -31,6 +31,7 @@
 #include "string_formatter.h"
 #include "string_input_popup.h"
 #include "translations.h"
+#include "try_parse_integer.h"
```
Comment (ZhilkinSerg) [inline src/worldfactory.cpp:34]:
```suggestion
```

File: tools/clang-tidy-plugin/UnsequencedCallsCheck.h [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
 #include <clang/ASTMatchers/ASTMatchFinder.h>
 #include <llvm/ADT/StringRef.h>
+
```
Comment (ZhilkinSerg) [inline tools/clang-tidy-plugin/UnsequencedCallsCheck.h:6]:
```suggestion
```

**Pull request merged**